packages/transaction-controller/src/constants.ts

```diff
@@ -118,3 +126,78 @@ export const VALID_UNAPPROVED_TRANSACTION_TYPES = [
   TransactionType.tokenMethodTransferFrom,
   TransactionType.contractInteraction,
 ];
+
+export interface SwapsTokenObject {
```

Comment:
As mentioned in another comment, shall we move these to `utils/swaps.ts` as it's all specific to swaps?